sdcm/db_log_reader.py

```diff
@@ -91,6 +96,10 @@ def _read_and_publish_events(self) -> None:
             if self._last_log_position:
                 db_file.seek(self._last_log_position)
             for index, line in enumerate(db_file, start=self._last_line_no + 1):
+                if len(line) > LOG_LINE_MAX_PROCESSING_SIZE:
+                    del line  # to avoid filling the memory when lot of logs is writen at the same time
+                    continue
```

Comment:
instead of `del line` and `continue`, I'd do `line = line[:LOG_LINE_MAX_PROCESSING_SIZE]`
did you try that?
So in that case we would still parse beginning of log line where most important info is stored (e.g. Scylla's log level, which can be ERROR)

Reply:
I didn't tried that

Reply:
looks good so far:
![image](https://github.com/scylladb/scylla-cluster-tests/assets/340979/4009e574-6cac-4d0d-83ef-af59fefc9a6b)

@roydahan seems like @soyacz idea is working nicely as well

Reply:
Cool, so I'm in favor of that.

Reply:
o.k. merging it, so we can have in the weekly tier1 runs